pkg/monitoring/metrics/virt-handler/migrationdomainstats/migrationstats_collector.go

```diff
@@ -53,41 +66,57 @@ var (
 	)
 )
 
-type migrationMetrics struct{}
+func SetupMigrationStatsCollector(vmiInformer cache.SharedIndexInformer) error {
+	var err error
```

Comment:
I think this var is not needed.

Reply:
It is needed to not override the global `migrationdomainstatsHandler` var